crypto/src/note.rs

```diff
@@ -47,6 +55,10 @@ pub enum Error {
     NoteTypeUnsupported,
     #[error("Note deserialization error")]
     NoteDeserializationError,
+    #[error("Encryption error")]
+    EncryptionError,
```

Comment:
Ideally we could have all encryptions be infallible and avoid this error variant.